example/src/ditto/config.yml

```diff
@@ -1,5 +1,5 @@
 projects:
-  - name: '[Demo] Onboarding Flow'
+  - name: 'Onboarding Flow'
     id: 61e72388365c930170607378
   - name: Ditto Component Library
-    id: ditto_component_library
\ No newline at end of file
+    id: ditto_component_library
```

Comment:
Can we update this to use the new `components: true` format like the config that was accidentally created in `example/ditto`?

Reply:
👍